activesupport/test/cache/cache_key_test.rb

```diff
@@ -78,6 +78,13 @@ def test_expand_cache_key_of_array_like_object
     assert_equal "foo/bar/baz", ActiveSupport::Cache.expand_cache_key(%w{foo bar baz}.to_enum)
   end
 
+  def test_validate_key
+    store = ActiveSupport::Cache::Store.new
+    assert_raises(ArgumentError) { store.send(:validate_key, nil) }
+    assert_raises(ArgumentError) { store.send(:validate_key, "") }
+    assert_nothing_raised { store.send(:validate_key, "foo_key") }
```

Comment:
I don't think we need to call private APIs to test that behavior.

Reply:
👍🏽 I've removed this.